mat/qr.go

```diff
@@ -87,18 +87,23 @@ func (qr *QR) Cond() float64 {
 // and upper triangular matrices.
 
 // RTo extracts the m×n upper trapezoidal matrix from a QR decomposition.
-// If dst is nil, a new matrix is allocated. The resulting dst matrix is returned.
-// RTo will panic if the receiver does not contain a factorization.
-func (qr *QR) RTo(dst *Dense) *Dense {
+//
+// If dst is empty, RTo will resize dst to be r×c. If dst is non-empty,
+// RTo will panic if dst is not r×c. RTo will also panic if the receiver
```

Comment:
"When dst is non-empty, RTo will panic if dst is not r×c."

Reply:
Done.